### PR TITLE
Using scrcpy instead of asm

### PR DESCRIPTION
**Pull request by Arne-B**
### Purpose of changes

Using a better maintained screen mirroring tool

### Types of changes
_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### How has this been tested?

Small scale manual testing has been performed.

### CODE REVIEW

**Comment by Arne-B** [comment]
https://github.com/budtmo/docker-android/issues/131

File: docker/Real_device [hunk 1]
```diff
@@ -69,9 +69,36 @@ RUN apt-get -qqy update && apt-get -qqy install --no-install-recommends \
     net-tools \
     ffmpeg \
     jq \
+    curl \
+    libavcodec-dev \
+    libavdevice-dev \
+    libavfilter-dev \
+    libavformat-dev \
```
Comment (budtmo) [inline docker/Real_device]: could you give more detail information as a comment like what kind of packages are those and why we need those?

File: docker/Real_device [hunk 2]
```diff
@@ -69,9 +69,36 @@ RUN apt-get -qqy update && apt-get -qqy install --no-install-recommends \
...
+    pkgconf \
+    libsdl2-dev \
  && apt clean all \
  && rm -rf /var/lib/apt/lists/*
 
+ARG SCRCPY_VER=1.10
+ARG SERVER_HASH="cbeb1a4e046f1392c1dc73c3ccffd7f86dec4636b505556ea20929687a119390"
+
+RUN PATH=$PATH:/usr/lib/jvm/java-1.8-openjdk/bin
+RUN git clone https://github.com/Genymobile/scrcpy.git
```
Comment (budtmo) [inline docker/Real_device]: I think we dont need to clone the repo or I miss something?

File: docker/Real_device [hunk 3]
```diff
@@ -156,7 +183,7 @@ EXPOSE 4723 6080 5555
 #===================
 # Run docker-appium
 #===================
-COPY sm/asm.jar /root/
+#COPY sm/asm.jar /root/
```
Comment (budtmo) [inline docker/Real_device]: I think we can delete the asm jar file that we dont need anymore.

File: docker/Real_device (hunk 1 above)
Comment (Arne-B) [reply docker/Real_device]: Its basically ffmpeg-dev, which is split in different packages. To be honest, we probably dont need all of them, but scrcpy build needed ffmpeg-dev, so i included all parts. Maybe i should check which parts are really used.

Comment (Arne-B) [reply docker/Real_device]: Its basically ffmpeg-dev, which is split in different packages. To be honest, we probably dont need all of them, but scrcpy build needed ffmpeg-dev, so i included all parts. Maybe i should check which parts are really used.

File: docker/Real_device (hunk 2 above)
Comment (Arne-B) [reply docker/Real_device]: Well, another option would be to get a prebuild scrcpy binary from somewhere.

File: docker/Real_device (hunk 3 above)
Comment (Arne-B) [reply docker/Real_device]: true!

File: docker/Real_device (hunk 1 above)
Comment (budtmo) [reply docker/Real_device]: yes, I agreed that we need to check which component that we really need, let me know if you need anything from me.

File: docker/Real_device (hunk 2 above)
Comment (budtmo) [reply docker/Real_device]: In my opinion it is better to use the stable version that scrcpy released which you did in the next line. Besides clone the repo just make the size bigger.

Comment (Arne-B) [reply docker/Real_device]: There is no Linux binary release. Only for the server jar. But fetching a released code zip and building from that instead might be a good idea stability wise.

File: docker/Real_device (hunk 1 above)
Comment (Arne-B) [reply docker/Real_device]: dependency('libavformat'),
        dependency('libavcodec'),
        dependency('libavutil'),

Are mentioned in the scrcpy meson file.

File: docker/Real_device (hunk 2 above)
Comment (Arne-B) [reply docker/Real_device]: We are now building from released zip, installing and removing sources afterwards.

**Pull request merged**